examples/polygon-p2p/src/chain_cfg.rs

```diff
@@ -3,7 +3,7 @@ use reth_chainspec::{
     once_cell_set, BaseFeeParams, Chain, ChainHardforks, ChainSpec, EthereumHardfork, ForkCondition,
 };
 use reth_discv4::NodeRecord;
-use reth_primitives::Head;
+use reth_ethereum_forks::Head;
```

Comment:
the head types makes this pr slightly inconvenient.
ideally we can find a better location for this or change/duplicate the type.